web/locales/it.json

```diff
@@ -1286,23 +1286,23 @@
             "zoneName": "El Salvador"
         },
         "SX": {
-            "countryName": "Sint Maarten",
-            "zoneName": "Olandese"
+            "countryName": "San Martino",
+            "zoneName": "Neerlandese"
         },
         "SY": {
             "zoneName": "Siria"
         },
         "SZ": {
-            "zoneName": "Swaziland"
+            "zoneName": "eSwatini"
```

Comment:
In the *La Farnesina* search I saw this named as `Regno di eSwatini`

<img width="462" alt="Screenshot 2020-10-12 at 18 17 15" src="https://user-images.githubusercontent.com/122919/95768944-64863e80-0cb7-11eb-93f6-355a9852d090.png">

Reply:
👌